### PR TITLE
task-1

### PR DESCRIPTION
**Pull request by ZiaurRehmanmalhi**
t1 add

### CODE REVIEW

File: task/task-1.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from utils.reader import read_file
+
+list_data = read_file()
+for data in list_data:
+    date = data.split(",")[0]
+    max_temp = data.split(",")[1]
+    min_temp = data.split(",")[3]
+    diffirance = int(max_temp) - int(min_temp)
```
Comment (Hamma111) [inline task/task-1.py]: wrong spelling of `difference`

File: task/task-1.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
```
Comment (Hamma111) [inline task/task-1.py]: never have two new lines spaces. only one line space is sufficient

File: utils/reader.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+def read_file():
+    file_values = []
+    with open("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv") as file_read:
+        file_contant = file_read.read()
+
+        for row in file_contant.split("\n")[1:-1]:
+            file_values.append(row)
+      
+        return file_values
```
Comment (Hamma111) [inline utils/reader.py]: remove one indentation before return statement
```
    return file_values
```

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: utils/reader.py (hunk 3 above)
Comment (ZiaurRehmanmalhi) [reply utils/reader.py]: ok sir

File: task/task-1.py (hunk 1 above)
Comment (ZiaurRehmanmalhi) [reply task/task-1.py]: ok change krta hn

File: task/task-1.py (hunk 2 above)
Comment (ZiaurRehmanmalhi) [reply task/task-1.py]: ok

File: utils/reader.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+def read_file():
+    file_values = []
+    with open("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv") as file_read:
+        file_contant = file_read.read()
+
+        for row in file_contant.split("\n")[1:-1]:
+            file_values.append(row)
+    return file_values
```
Comment (Hamma111) [inline utils/reader.py:8]: one line space before every return statement

**Review by Hamma111 — APPROVED** [review]

File: utils/reader.py (hunk 4 above)
Comment (ZiaurRehmanmalhi) [reply utils/reader.py:8]: ok

**Pull request merged**